src/Hash.cc

```diff
@@ -184,6 +185,29 @@ HashKey::HashKey(const void* arg_key, size_t arg_size, hash_t arg_hash, bool /*
 	key = (char*)arg_key;
 	}
 
+HashKey::HashKey(const HashKey& other) : HashKey(other.key, other.size, other.hash) { }
+
+HashKey::HashKey(HashKey&& other) noexcept
```

Comment:
The move constructor can't be because it explicitly sets certain fields in the other object to certain values to make sure it's in a valid-but-invalid state.